.moban.cd/moban.yml

```diff
@@ -15,9 +15,10 @@ keywords:
   - jinja2
   - moban
 dependencies:
-  - ruamel.yaml
+  - ruamel.yaml==0.15.86
   - jinja2>=2.7.1
-  - lml>=0.0.7
+  - lml>=0.0.9
+  - appdirs==1.4.3
```

Comment:
We can probably allow lower, but this is OK with most updated distros, as 1.4.3 is quite old.

e.g. https://build.opensuse.org/package/show/devel:languages:python/python-appdirs

Reply:
would it be too much picky when it comes to OpenSuse?